src/output.ts

```diff
@@ -16,9 +24,33 @@ export function outputUser(message: string) {
 }
 
 export function outputAi(message: string) {
+  if (syntaxHighlighting) {
+    outputSyntaxHighlighting(message);
+    return;
+  }
+
   console.log(chalk.cyan('ai:', message));
 }
 
+export function outputSyntaxHighlighting(message: string) {
+  const splitted = splitCodeBlocks(message);
+  chalk.cyan('ai: ');
+  splitted.forEach((segment) => {
+    if (segment.match(codeBlockAnnotationsRegExp)) {
+      const lang = segment.match(langRegExp)?.[1];
+      const stripped = stripCodeblockAnnotations(segment);
+      console.log(
+        highlight(stripped, {
+          language: lang ?? 'plaintext',
```

Comment:
Do we want to default to `plaintext`? How would that affect language aotodetection.